### ISSUE TITLE
Members do not get welcome email when joining a free tier

### ISSUE THREAD

**Issue by Abekonge** [opencollective/opencollective#7110]
**Describe the bug**
When joining a member tier with a free option, you do not receive the welcome email. Possibly related to #6774, 

**To Reproduce**
1. Go to https://opencollective.com/kollektiv-email
2. Click on third tier, the one with the following description `Bliv medlem af Kollektiv Email — betal hvad du kan (også 0 kroner). Det er muligt for alle indkomstniveauer at være med.` It's a membership tier with a free option
3. Select `Free`
4. Click `Your info`
5. Add email, catpcha.
6. Click `Make contribution`

**Expected behavior**
The member should receive the welcome-email.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS Sonoma
 - Browser: Firefox
 - Version: 119.0.1 (64-bit)

**Comment by Betree** [comment]
This is indeed related to https://github.com/opencollective/opencollective/issues/6774, but we could send the contribution confirmation email (aka. "thank you" email) without creating the member entry.

**Comment by Abekonge** [comment]
> This is indeed related to #6774, but we could send the contribution confirmation email (aka. "thank you" email) without creating the member entry. We probably need to look at and adapt the template though.

Yes that makes sense. I'm up for another issue if this is suitable - and my current free tier member-flow sort of depends on this 😄(although I should probably wait for my first issue #7077 to be reviewed 🤞)
